recipes/funcparserlib/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "funcparserlib" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b7992eac1a3eb97b3d91faa342bfda0729e990bd8a43774c1592c091e563c91d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

should it be ``noarch``?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

---

The library uses 2-3 in its setup.py, so we can't use noarch.